web/web-backend/src/main/kotlin/com/testerum/web_backend/services/runner/tree/RunnerTreeBuilder.kt

```diff
@@ -0,0 +1,233 @@
+package com.testerum.model.runner.tree
+
+import com.testerum.file_service.caches.resolved.FeaturesCache
+import com.testerum.model.feature.hooks.HookPhase
+import com.testerum.model.infrastructure.path.HasPath
+import com.testerum.model.infrastructure.path.Path
+import com.testerum.model.runner.tree.id.RunnerIdCreator
+import com.testerum.model.runner.tree.model.RunnerBasicStepNode
+import com.testerum.model.runner.tree.model.RunnerComposedStepNode
+import com.testerum.model.runner.tree.model.RunnerFeatureNode
+import com.testerum.model.runner.tree.model.RunnerHooksContainerNode
+import com.testerum.model.runner.tree.model.RunnerHooksNode
+import com.testerum.model.runner.tree.model.RunnerNode
+import com.testerum.model.runner.tree.model.RunnerParametrizedTestNode
+import com.testerum.model.runner.tree.model.RunnerRootNode
+import com.testerum.model.runner.tree.model.RunnerScenarioNode
+import com.testerum.model.runner.tree.model.RunnerStepNode
+import com.testerum.model.runner.tree.model.RunnerTestNode
+import com.testerum.model.runner.tree.model.RunnerUndefinedStepNode
+import com.testerum.model.runner.tree.model.enums.HooksType.AFTER_ALL_HOOKS
+import com.testerum.model.runner.tree.model.enums.HooksType.AFTER_EACH_HOOKS
+import com.testerum.model.runner.tree.model.enums.HooksType.AFTER_HOOKS
+import com.testerum.model.runner.tree.model.enums.HooksType.BEFORE_ALL_HOOKS
+import com.testerum.model.runner.tree.model.enums.HooksType.BEFORE_EACH_HOOKS
+import com.testerum.model.step.BasicStepDef
+import com.testerum.model.step.ComposedStepDef
+import com.testerum.model.step.StepCall
+import com.testerum.model.step.UndefinedStepDef
+import com.testerum.model.test.TestModel
+import com.testerum.model.test.scenario.Scenario
+import com.testerum.model.tests_finder.ScenariosTestPath
+import com.testerum.model.util.new_tree_builder.ContainerTreeNode
+import com.testerum.model.util.new_tree_builder.PathBasedTreeBuilder
+import com.testerum.model.util.new_tree_builder.TreeNode
+import com.testerum.model.util.new_tree_builder.TreeNodeFactory
+
+class RunnerTreeBuilder(val featureCache: FeaturesCache) {
```

Comment:
We need to delete this file, right? It looks like you moved the old file here, for a backup, and then forgot to delete it.